app/controllers/api_controller.rb

```diff
@@ -20,6 +20,7 @@ def respond_to_format(response_data, filename)
         send_data export_organizations_as_docx(response_data), filename: "#{filename}.docx" 
       end
       format.markdown {render text: export_organizations_as_markdown(response_data)}
+      format.any {  raise ActionController::RoutingError.new('Not Found') }
```

Comment:
It looks like there are two spaces before `raise`, which is probably why this was commented on.